### ISSUE TITLE
Improvement for Challenge Mechanics

### ISSUE THREAD

**Issue by olgamik** [graphprotocol/mission-control-curator#378]
Hello, I think the owner of the project on Everest that was challenged should has an abillity to place the answer to description of the challenge. People will read both the description and the answer and it will be easier to decide who is right

**Comment by yanivtal** [comment]
Yes sounds like we need better communication. Maybe a comments plugin?

**Comment by olgamik** [comment]
Yes, it will be even better with comments plugin. This plugin can be used for regular projects as well, not only for challenged.  For example, if some information about the project has changed, then users will be able to indicate this to the owner of the project on Everest and he will fix it
